src/lib/core/DataModelTypes.h

```diff
@@ -58,6 +58,8 @@ constexpr EndpointId kRootEndpointId    = 0;
 constexpr ListIndex kInvalidListIndex   = 0xFFFF; // List index is a uint16 thus 0xFFFF is a invalid list index.
 constexpr KeysetId kInvalidKeysetId     = 0xFFFF;
 
+constexpr uint64_t kInvalidIcId = 0;
```

Comment:
What is IcId and why is this an invalid value for it?  (I see it in the XML that got added, but not in the spec....)

Reply:
an ICid is an intermittently  connected device identifier. The goal of this "token" is for a controller to be able to identify an ICD without needing a case session to be completed. This identifier is going to be used in the checkin message that is being defined.

This identifier is being designed by @robszewczyk  and is not yet defined the spec. For developpement purposes, we are going to use a hash of the nodeid. 

Most of the developement done is not yet defined by the spec.